package.json

```
@@ -12,10 +12,11 @@
     "private": true,
     "dependencies": {
         "@angular/animations": "~9.0.5",
-        "@angular/cdk": "^9.1.1",
+        "@angular/cdk": "^9.2.0",
```

The '^' or '~' sign indicates that npm will take the latest minor or patch of a specific version. Perhaps it's better to clearly indicate this using 'x' like so:

```suggestion
        "@angular/cdk": "^9.x.x",
```

This could be done for all dependencies in this file.